techlibs/intel/synth_intel.cc

```diff
@@ -147,8 +147,12 @@ struct SynthIntelPass : public ScriptPass {
 
 		if (!design->full_selection())
 			log_cmd_error("This command only operates on fully selected designs!\n");
-		if (family_opt != "max10" && family_opt != "a10gx" && family_opt != "cyclonev" && family_opt != "cycloneiv" &&
-		    family_opt != "cycloneive" && family_opt != "cyclone10")
+		if (family_opt != "max10" &&
+		    family_opt != "a10gx" &&
+		    family_opt != "cyclonev" &&
+		    family_opt != "cycloneiv" &&
+		    family_opt != "cycloneive" &&
+		    family_opt != "cyclone10")
 			log_cmd_error("Invalid or not family specified: '%s'\n", family_opt.c_str());
```

Comment:
Nitpick - while you are touching this code, can you fix "not family" to be "no family" here?

Reply:
I found another instance of the same typo, which I also fixed.